packages/extras/src/lib/components/Instancing/InstancedMesh.svelte

```diff
@@ -17,6 +18,9 @@
   export let range = 1000
   export let update = true
 
+  // forward ref binding
+  export let ref: InstancedMesh | undefined = undefined
```

Comment:
Because the actual instance is not going to change, it's more comfortable to just instantiate one here, so we don't have to deal with `| undefined` upstream when binding.

```suggestion
  export const ref = new InstancedMesh(null as any, null as any, 0)
```

Reply:
Good point!